### ISSUE TITLE
Request for releasing organization's name.

### ISSUE THREAD

**Issue by crossRT** [DeVinciCode/devincicode.github.io#1]
Hi there, I found out that your website is down for quite some time and so unluckily we have a team here that name 'DevinciCode' as well. 
It will be nice to have this name as our GitHub organization name. 
We sincerely hope that you could release this name for us. Let us know asap, and thank you for your consideration.

**Comment by MarcAudefroy** [comment]
Hi crossRT,
No problem.

**Comment by crossRT** [comment]
Thank you so much @MarcAudefroy, it means so much to our team.
I will check out the name availability or you can drop me a message to crossRT@gmail.com
Once again, thank you. =)

**Comment by MarcAudefroy** [comment]
I send you an invite to join DeVinciCode organization.

**Comment by crossRT** [comment]
yeah. I received, but it's an invitation to this repo, instead of the organization.

**Comment by MarcAudefroy** [comment]
whoops. Resend it
